parsers.c

```diff
@@ -1540,6 +1539,14 @@ static jb_err header_tagger(struct client_state *csp, char *header)
             continue;
          }
 
+         if (list_contains_item(csp->action->multi[ACTION_MULTI_SUPPRESS_TAGGER], b->name))
```

Comment:
Is there a O(1) or O(log(n)) lookup?

Reply:
0(n)

Reply:
No tareces of the sort() calls in the code - I assume there are no 0(log(n)) lookups: even map() is implemented as linked list. The only hash_string() usage is to use switch/case over string.